lib/core.ps1

```diff
@@ -1300,6 +1300,16 @@ Optimize-SecurityProtocol
 # Load Scoop config
 $configHome = $env:XDG_CONFIG_HOME, "$env:USERPROFILE\.config" | Select-Object -First 1
 $configFile = "$configHome\scoop\config.json"
+# Check if it's the expected install path for scoop: <root>/apps/scoop/current
+$coreRoot = Split-Path $PSScriptRoot
+$pathExpected = ($coreRoot -replace '\\','/') -like '*apps/scoop/current*'
+if ($pathExpected) {
+    # Portable config is located in root directory: .\current\scoop\apps\<root>
```

Comment:
```suggestion
    # Portable config is located in root directory: .\current\scoop\<root>
```

`$coreRoot` is the `current` dir of scoop, and three `..` go to `$scoopdir`. A typo?

Reply:
No, it's not.

- `$coreRoot\` - `.\current\`
- `..\..\` - `scoop\apps\`
- `..\config.json` - `<root>\config.json`.

Reply:
I mean the config file this loaded is under `scoop`, not `apps`.

Reply:
Aha, yes, this is what I mean.